src/components/connect.js

```diff
@@ -164,7 +164,14 @@ function connect(mapPropsToRequestsToProps, defaults, options) {
         meta: {}
       },
       defaults,
-      parent ? { comparison: parent.comparison } : {},
+      parent ? {
+        fetch: parent.fetch,
+        buildRequest: parent.buildRequest,
+        handleResponse: parent.,
```

Comment:
ugh, not sure how that happened... must have deleted it between running tests and committing. fixed